lib/rainforest/cli/runner.rb

```diff
@@ -14,7 +14,9 @@ def run
         CSVImporter.new(options.import_name, options.import_file_name, options.token).import
       end
 
-      post_opts = make_create_run_options
+      app_source_url = options.app_source_url ? upload_app(options.app_source_url) : options.app_source_url
```

Comment:
Seems like this would logically fit better under `make_create_run_options`. That way we can remove the new argument too.